server/embed/config.go

```diff
@@ -216,10 +223,27 @@ type Config struct {
 	DNSCluster            string `json:"discovery-srv"`
 	DNSClusterServiceName string `json:"discovery-srv-name"`
 	Dproxy                string `json:"discovery-proxy"`
-	Durl                  string `json:"discovery"`
-	InitialCluster        string `json:"initial-cluster"`
-	InitialClusterToken   string `json:"initial-cluster-token"`
-	StrictReconfigCheck   bool   `json:"strict-reconfig-check"`
+
+	EnableV2Discovery bool   `json:"enable-v2-discovery"`
```

Comment:
Could we maybe consider a substruct?

Reply:
Yes, It's a good point. I thought about it as well.

I added a struct [DiscoveryConfig](https://github.com/ahrtr/etcd/blob/v3_discovery/server/etcdserver/api/v3discovery/discovery.go#L57), and it is used in [ServerConfig](https://github.com/ahrtr/etcd/blob/v3_discovery/server/config/config.go#L37), but not used here. 

So I will use struct [DiscoveryConfig](https://github.com/ahrtr/etcd/blob/v3_discovery/server/etcdserver/api/v3discovery/discovery.go#L57) here as well.

Reply:
For example DiscoveryConfig?

Reply:
Fixed.